### PR TITLE
Add PostgreSQL SSL mode in README.md

### PR DESCRIPTION
**Pull request by kakisoft**
In README, there is no statement about PostgreSQL in SSL mode.
So, I would like to suggest to add it.

Refer to:
https://github.com/k1LoW/tbls/issues/300

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ Using docker image.
 $ docker run --rm -v $PWD:/work k1low/tbls doc postgres://dbuser:dbpass@hostname:5432/dbname
 ```
 
+In SSL mode, add "?sslmode=disable"
```
Comment (k1LoW) [inline README.md:64]: I think you need to disable ssl mode when the PostgreSQL server does not support SSL connections.

**Comment by k1LoW** [comment]
@kakisoft Thank you!!

File: README.md (hunk 1 above)
Comment (kakisoft) [reply README.md]: Sorry, I didn't explain enough.
I just would like you to add this statement, not replace.

So, This is what I meant.

Before:
image

After:
image

But, I made a mistake first pull request.
I'm sorry for confusing you.

I tried to add the statement under PostgreSQL connection.
But, first pull request has an error, I added the statement under Quick Start.

So, I fixed it and sent you pull request again.

Comment (k1LoW) [reply README.md]: Why `In SSL mode, add "?sslmode=disable"` ?

SSL mode is a connection method between client and server. It is not a setting that should always be disabled.

Comment (kakisoft) [reply README.md]: Sorry, my phrasing was bad.
I will rephrase it.

## Before:
```
In SSL mode, add "?sslmode=disable"
```

## After:
```
When you want to disable SSL mode, add "?sslmode=disable"
```

**Comment by k1LoW** [comment]
Thank you!

**Pull request merged**